_posts/2019-05-20-foreman-community-newsletter---may-2019.md

```diff
@@ -0,0 +1,175 @@
+Foreman documentation followed by a discussion around how we can make it better as a community. We did have @lzap, at the meetup as he was in Pune at the same time, to speak about the importance and history of open source! s
+
+* [Foreman templates UI](https://community.theforeman.org/t/foreman-templates-ui-deep-dive/13116)
+
+@Ondrej_Prazak was kind enough to educate the community about the long awaited deep
+drive about foreman templates and the new UI for the plugin. He described in brief about
+the plugin after which he also mentioned the plan to support the current workflows with the new UI. Feel free to checkout his [pull request](https://github.com/theforeman/foreman_templates/pull/109) and leave your thought/comments on it.
+
+* [Redesigning the javascript stack](https://community.theforeman.org/t/deep-dive-rfc-redesigning-the-javascript-stack/13629)
+
+@sharvit had spoken about the various issues that one faces with the current Javascipt
+stack. His deep drive basically explains the need of having to restructure the Javascript
+stack altogether. Good part about his deep drive was that he not only pointed out the issues
+with the old Javascript stack but also provided a solutions for the same. One can read his
+thoughts on the [forum](https://community.theforeman.org/t/redesigning-the-javascript-stack/13470)
+
+* [Community Demo #58](https://community.theforeman.org/t/foreman-community-demo-58)
+
+Hosting the community demo for the first time, I could not be more overwhelmed to have about
+seven talks for the demo. The demo illustates few of the highlighted features of foreman 1.22.0 like global parameters with types and GraphQL console.
+
+* [Community Demo #59](https://community.theforeman.org/t/foreman-community-demo-59)
+
+Finally! we have the result of the Foreman community survey. @Gwmngilfen demostated his skill in terms of graphs and bars to display results of the survey.
+
+* [Community Demo #60](https://community.theforeman.org/t/foreman-community-demo-60)
+
+@kgaikwad was kind enough to educate us with the working of GCE provisioning while we had @lzap to talk about the unattended action issues that were bugging him. Few more detailed updates by @Gwmngilfen about the community survey. I guess it is time to use the data from the demo and make some decisions about them! If you have any comments or concerns about the
+survey please reach out to me!
+
+* [Community Demo #61](https://community.theforeman.org/t/foreman-community-demo-61)
+
+Woah! finally we have Foreman appearance at Red Hat summit again. Tomer spoke about the
+presence of Foreman project at the community booth at the summit. Also, @Ondrej_Prazak, one of our most consistant speakers at the community demo, spoke about the Openscap deployment by Ansible. Till then it Openscap deployment was only possible though puppet but now we know, we can do it through Ansible too!
+
+
+### Plugin news
+
+Updated plugins:
```

Comment:
Might be good to make sure there weren't new versions released since you last updated the list